desktop/apps/buy_art/stylesheets/buttons.styl

```diff
@@ -0,0 +1,288 @@
+// @require '../../stylus_lib'
+// @require '../../../../node_modules/artsy-ezel-components/spinners'
+// @require '../../follow_button'
+
+.circle-button
```

Comment:
These look like duplicate rules, why not just import these from the existing stylesheets?